### PR TITLE
Enable tensorflow keras model examples using new API

### PR DESCRIPTION
**Pull request by lvliang-intel**
## Type of Change

feature
API not changed 

## Description

detail description 
JIRA ticket: ILITV-2598

## Expected Behavior & Potential Risk

The left keras models are all re-enabled with new api.

## How has this PR been tested?

Pre-CI and extension test.

## Dependency Change?

No.

### CODE REVIEW

File: examples/tensorflow/image_recognition/keras_models/inception_resnet_v2/quantization/ptq/README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+This document is used to enable Tensorflow Keras model inception_resnet_v2 quantization and benchmark using Intel® Neural Compressor.
 This example can run on Intel CPUs and GPUs.
 
 
-## Prerequisite
+# Prerequisite
+
+## 1. Environment
 
 ### 1. Installation
```
Comment (chensuyue) [inline examples/tensorflow/image_recognition/keras_models/inception_resnet_v2/quantization/ptq/README.md:12]: It seems clearer to remove the `1.` of sub title.

File: examples/README.md [hunk 2]
```diff
@@ -103,13 +103,13 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td>*VGG16</td>
```
Comment (chensuyue) [inline examples/README.md:103]: Please remove the `*` of the example after API migrate.

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4167/)

**Review by zehao-intel — APPROVED** [review]

**Review by Spycsh — APPROVED** [review]

**Comment by lvliang-intel** [comment]
extension test: https://inteltf-jenk.sh.intel.com/blue/organizations/jenkins/intel-lpot-validation-top-mr-extension/detail/intel-lpot-validation-top-mr-extension/4198

**Comment by lvliang-intel** [comment]
extension test:
https://inteltf-jenk.sh.intel.com/blue/organizations/jenkins/intel-lpot-validation-top-mr-extension/detail/intel-lpot-validation-top-mr-extension/4330

**Comment by lvliang-intel** [comment]
Extension test result:
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4330/artifact/report.html

The root cause of the performance gap is that the original eval_func for these Keras models is built-in which using the session_run.  Now we defined user eval_func which using the saved_model signatures infer to run inference.
This is for keeping consistency with the previous enabled Keras models and we prefer using this way.

File: examples/README.md [hunk 3]
```diff
@@ -154,46 +154,46 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td><a href="./tensorflow/image_recognition/tensorflow_models/efficientnet-b0/quantization/ptq">ckpt</a></td>
   </tr>
   <tr>
-    <td>*MNIST </td>
+    <td>MNIST </td>
     <td>Image Recognition</td>
     <td>Quantization-Aware Training</td>
-    <td><a href="https://github.com/intel/neural-compressor/tree/old_api_examples/examples/tensorflow/image_recognition/keras_models/mnist/quantization/qat">keras</a></td>
+    <td><a href="./examples/tensorflow/image_recognition/keras_models/mnist/quantization/qat">keras</a></td>
```
Comment (chensuyue) [inline examples/README.md:160]: fix the link .

File: examples/tensorflow/image_recognition/keras_models/resnet101/quantization/ptq/README.md [hunk 4]
```diff
@@ -1,23 +1,25 @@
...
 pip install neural-compressor
 ```
-### 2. Install Tensorflow
+### Install Tensorflow
 ```shell
 pip install tensorflow
 ```
-> Note: Validated TensorFlow [Version](/docs/source/installation_guide.md#validated-software-environment).
-### 3. Install Intel Extension for Tensorflow
+> Note: Supported Tensorflow [Version](../../../../../../../README.md).
```
Comment (chensuyue) [inline examples/tensorflow/image_recognition/keras_models/resnet101/quantization/ptq/README.md:21]: This link is invalid, we need to use this one:   
> Note: Validated TensorFlow [Version](/docs/source/installation_guide.md#validated-software-environment).

File: examples/tensorflow/image_recognition/keras_models/resnet50/quantization/ptq/README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+### Installation
+```shell
+# Install Intel® Neural Compressor
+pip install neural-compressor
+```
+### Install Intel Tensorflow
+```shell
+pip install intel-tensorflow
+```
+> Note: Supported Tensorflow [Version](../../../../../../../README.md).
```
Comment (chensuyue) [inline examples/tensorflow/image_recognition/keras_models/resnet50/quantization/ptq/README.md:21]: > Note: Validated TensorFlow [Version](/docs/source/installation_guide.md#validated-software-environment).

File: examples/tensorflow/image_recognition/keras_models/resnet50_fashion/quantization/ptq/README.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+### Installation
+```shell
+# Install Intel® Neural Compressor
+pip install neural-compressor
+```
+### Install Intel Tensorflow
+```shell
+pip install intel-tensorflow
+```
+> Note: Supported Tensorflow [Version](../../../../../../../README.md#supported-frameworks).
```
Comment (chensuyue) [inline examples/tensorflow/image_recognition/keras_models/resnet50_fashion/quantization/ptq/README.md:21]: > Note: Validated TensorFlow [Version](/docs/source/installation_guide.md#validated-software-environment).

File: examples/tensorflow/image_recognition/keras_models/vgg16/quantization/ptq/README.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+### Installation
+```shell
+# Install Intel® Neural Compressor
+pip install neural-compressor
+```
+### Install Intel Tensorflow
+```shell
+pip install intel-tensorflow
+```
+> Note: Supported Tensorflow [Version](../../../../../../../README.md#supported-frameworks).
```
Comment (chensuyue) [inline examples/tensorflow/image_recognition/keras_models/vgg16/quantization/ptq/README.md:21]: > Note: Validated TensorFlow [Version](/docs/source/installation_guide.md#validated-software-environment).

File: examples/tensorflow/image_recognition/keras_models/vgg19/quantization/ptq/README.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+### Installation
+```shell
+# Install Intel® Neural Compressor
+pip install neural-compressor
+```
+### Install Intel Tensorflow
+```shell
+pip install intel-tensorflow
+```
+> Note: Supported Tensorflow [Version](../../../../../../../README.md#supported-frameworks).
```
Comment (chensuyue) [inline examples/tensorflow/image_recognition/keras_models/vgg19/quantization/ptq/README.md:21]: > Note: Validated TensorFlow [Version](/docs/source/installation_guide.md#validated-software-environment).

**Pull request merged**